index.js

```diff
@@ -119,6 +119,12 @@ class RemoteBrowser extends EventEmitter {
           debug(`Redirected to ${frame.url()}`, 'info');
         });
 
+        if (this.CLEAR_COOKIES) {
+          const client = await target.createCDPSession();
```

Comment:
@dshster, it is strange that you creates the session but does not shut it down. Should it be destroyed or something?

E.g. I've found this method: https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#cdpsessiondetach

Reply:
The CDP session will be destroyed when the page is closed, but I have added a force detach.